docs/02_Hardware_References/02_MCUs.md

```diff
@@ -0,0 +1,141 @@
+
+# ESP32
+
+The UDOO KEY is powered by Espressif’s **ESP32-WROVER-E** MCU module, a powerful and versatile platform for Edge AI applications, ranging from low-power sensor networks to the most demanding tasks, such as voice encoding, music streaming and MP3 decoding. This module is provided with a PCB antenna.  
+This fully programmable microcontroller is based on a **dual-core Xtensa 32-bit LX6**, with a 16 MB internal SPI flash memory, an additional 8 MB SPI Pseudo Static RAM (PSRAM), Wi-Fi and Bluetooth.
+
+The core of the module is the **ESP32-D0WD-V3** chip, which is designed to be scalable and adaptive. It features two CPU cores that can be individually controlled, and the CPU clock frequency is adjustable from 80 MHz to 240 MHz. The chip is also provided with a low-power co-processor that can be used instead of the CPU to save power while performing tasks that do not require much computing power, such as perihperals monitoring.
+
+Thanks to the ESP32 on board, you get full Wi-Fi 802.11b/g/n connectivity, Bluetooth and BLE v4.2. This ensures the UDOO KEY’s extreme versatility: the Wi-Fi allows to set up direct connection to the internet through a Wi-Fi router, while using the Bluetooth users can connect to their phones or broadcast low energy beacons for its detection.
+
+Mind you: the ESP32 on board is not just a Wi-Fi, BT & BLE module. On the contrary, it gives you full access to the firmware of ESP32 and you can program it as you please. 
```

Comment:
s/Mind you/Note/

Reply:
On the contrary XXX

Reply:
Solved with bbe86fa1935fee4666925dabb13f68beaf635452